src/features/island/fruit/FruitPatch.tsx

```diff
@@ -28,7 +28,12 @@ export const FruitPatch: React.FC<Props> = ({ id }) => {
 
   const gameState = useSelector(gameService, (state) => ({
     fruit: state.context.state.fruitPatches[id]?.fruit,
-    isPlaying: state.matches("playing") || state.matches("autosaving"),
+    isPlaying:
+      state.matches("playing") ||
+      state.matches("playingGuestGame") ||
+      state.matches("playingFullGame") ||
+      state.matches("autosaving") ||
+      state.matches("guestAutosaving"),
```

Comment:
This is getting a bit hectic. Even having 2 states for an `isPlaying` mode was a bit of a code smell. Having 6 states feels like we are stretching the definition of playing

Reply:
We have consoldated `loadingGuestGame`, `loadingFullGame` into `loading` and `guestAutosaving` into `autosaving`.